app/graphql/resolvers/customer/customer.js

```diff
@@ -8,17 +9,31 @@ import {
 import { transformOrganisationAuthorisationToCustomerBusinessPermissionLevel } from '../../../transformers/rural-payments-portal/permissions.js'
 
 export const Customer = {
-  async business ({ customerId }, { sbi }, { dataSources }) {
```

Comment:
this is supposed to be crn, not customerId

Reply:
I haven't — I've changed where we're getting the data from. Before, the `customerId` was coming from the parent object because it was being resolved at the top-level. But now that doesn't exist there, but the CRN does. So now, rather than having the `customerId` passed from the parent, I'm using the `crn` to look-up the `customerId` in this resolver itself. Same in the other example.

Reply:
Looks like this is fixed though on line 22